src/utils/appUtils.ts

```diff
@@ -140,3 +140,19 @@ export const convertToSentenceCase = (str: string) =>
   str.split(" ").length > 1
     ? str.charAt(0).toUpperCase() + str.slice(1).toLowerCase()
     : str
+
+/**
+ * Converts camel case string to shish kabob case
+ *
+ * e.g. camelToShishKabobCase("RecapPul")
+ *        => "recap-pul"
+ *      camelToShishKabobCase("firstCharCanBeLowerCase")
+ *        => "first-char-can-be-lower-case"
+ */
+export const convertCamelToShishKabobCase = (str: string) =>
+  str
+    // Change capital letters into "-{lowercase letter}"
+    .replace(/([A-Z])/g, (c, p1, i) => {
```

Comment:
What's the `p1` and `i` for?

Reply:
i think `i` is the index needed to skip the first character, and p1 is a placeholder arg

Reply:
Just updated these var names for clarity